README.md

```diff
@@ -12,6 +12,9 @@ Before you begin, make sure you have the following installed:
 - Git
 - Make
 - Python (check the `pyproject.toml` file for the required version)
+- Poetry
```

Comment:
per @debugmiller 's comment, it seems poetry is not really needed but it's one of the options?

Reply:
True, but see [this slack comment](https://panther-labs.slack.com/archives/C06VCSMHZE1/p1720543884338609?thread_ts=1720464728.444789&cid=C06VCSMHZE1)

Reply:
Ah! Makes sense to me :)